tests/integration/iceberg/test_table_load.py

```diff
@@ -82,3 +82,40 @@ def test_daft_iceberg_table_renamed_column_pushdown_collect_correct(local_iceber
     iceberg_pandas = tab.scan().to_arrow().to_pandas()
     iceberg_pandas = iceberg_pandas[["idx_renamed"]]
     assert_df_equals(daft_pandas, iceberg_pandas, sort_key=[])
+
+
+@pytest.mark.integration()
```

Comment:
add 2 tests:
- select only the partition column with filter on it
- select only the partition column with filter on another column

Reply:
Added the two requested tests:

1. This fails, because reading just the partition column only still fails after this PR (we dont have a great way of "reading [] columns from a file" at the moment
2. This passes